doc/config/modules.conf

```diff
@@ -51,6 +51,7 @@
 server.modules = (
 #  "mod_rewrite",
   "mod_access",
+#  "mod_openssl",
```

Comment:
I do not think maintaining a complete list here is necessary.

Besides, lighttpd supports multiple TLS modules as options.  If one is listed, then all should be listed.  Instead, I think none should be listed here.  On the other hand, if there are uncommented `ssl.*` directives in the examples, there should probably be an uncommented `server.modules` including a TLS module.

lighttpd supports and ignores duplicated modules in `server.modules` since lighttpd 1.4.56, so this can be located near examples: `server.modules += ( "mod_openssl" )`.  `server.modules += ("mod_openssl")` should be added to the commented out example in the sample `lighttpd.conf`.

Reply:
I have added `server.modules += ( "mod_openssl" )` in the commented out section of sample `lighttpd.conf` as per your suggestion and removed `mod_openssl` from `modules.conf`.